.docker/docker-compose.ci.yml

```diff
@@ -11,3 +11,4 @@ services:
       - GITHUB_TOKEN=${GITHUB_TOKEN}
       - JEKYLL_ENV=${JEKYLL_ENV}
       - GITHUB_REPO=${GITHUB_REPO}
+      - GIT_DEPLOY_DIR="/srv/jekyll/_site"
```

Comment:
Hm, it feels wrong to provide detailed configuration for the insides of the `jekyll-plantuml` container here. Can't `JEKYLL_PLANTUML` provide this variable within the container instead? Since the container doesn't provide a way to configure the location of the `_site` folder, it makes little sense to provide that configuration to the outside.